### PR TITLE
[Build] Pass pkgconfig args when compiling ClangModule

### PR DESCRIPTION
**Pull request by aciidb0mb3r**
_No test case yet_

### CODE REVIEW

File: Sources/Build/Command.link(ClangModule).swift [hunk 1]
```diff
@@ -28,7 +28,7 @@ extension Command {
             let buildMeta = ClangModuleBuildMetadata(module: module, prefix: prefix, otherArgs: [])
             objects += buildMeta.objects
             inputs += buildMeta.inputs
-            linkFlags += buildMeta.linkDependenciesFlags + module.languageLinkArgs
+            linkFlags += buildMeta.linkDependenciesFlags + (try module.pkgConfigArgs().libs)
```
Comment (neonichu) [inline Sources/Build/Command.link(ClangModule).swift]: I think this should be in the `.Library(.Dynamic)` case, as there is `--static` for static linking once we implement that.

**Comment by ddunbar** [comment]
I think we need to find a way to get a test case for this.

**Comment by aciidb0mb3r** [comment]
@swift-ci please test

**Comment by aciidb0mb3r** [comment]
@swift-ci please test

**Comment by aciidb0mb3r** [comment]
CI is down, tested locally

**Pull request merged**